src/zenml/integrations/aws/flavors/sagemaker_orchestrator_flavor.py

```diff
@@ -0,0 +1,90 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Amazon SageMaker orchestrator flavor."""
+
+from typing import TYPE_CHECKING, Optional, Type
+
+from zenml.config.base_settings import BaseSettings
+from zenml.integrations.aws import AWS_SAGEMAKER_STEP_OPERATOR_FLAVOR
+from zenml.orchestrators.base_orchestrator import BaseOrchestratorFlavor
+from zenml.step_operators.base_step_operator import BaseStepOperatorConfig
+
+if TYPE_CHECKING:
+    from zenml.integrations.aws.orchestrators import SagemakerOrchestrator
+
+
+class SagemakerOrchestratorSettings(BaseSettings):
+    """Settings for the Sagemaker orchestrator."""
+
+
+class SagemakerOrchestratorConfig(  # type: ignore[misc] # https://github.com/pydantic/pydantic/issues/4173
+    BaseStepOperatorConfig, SagemakerOrchestratorSettings
+):
+    """Config for the Sagemaker orchestrator.
+    Attributes:
+        instance_type: The instance type to use for the processing job.
+        bucket: Name of the S3 bucket to use for storing artifacts
+            from the job run. If not provided, a default bucket will be created
+            based on the following format: "sagemaker-{region}-{aws-account-id}".
+    """
+
+    # 'ml.t3.medium' is the default instance for Sagemaker processing jobs
+    instance_type: str = "ml.t3.medium"
```

Comment:
Rename to default_instance_type as its used in every step by default unless overriden by settings?